tiled/schemas/service_configuration.yml

```diff
@@ -378,3 +378,10 @@ properties:
       ssl_ciphers:
         type: string
         description: Ciphers to use (see stdlib ssl module's). Default TLSv1.
+  metrics:
+    type: object
+    properties:
+      prometheus:
+        type: boolean
+        description: |
+          Enable/Disable prometheus metrics
```

Comment:
Can you note that it is `false` (off) by default and that, if set to `true` (on) the `PROMETHEUS_METRICS_DIR` environment variable must be set to the path of a writable directory?